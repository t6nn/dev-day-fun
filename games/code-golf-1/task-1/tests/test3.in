99879874537878 93409348894 2934434 34343555 234343332 234656655