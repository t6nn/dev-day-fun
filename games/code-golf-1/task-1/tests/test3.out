87873547897899 49884390439 4344392 55534343 233343432 556656432